projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/credentials/KerberosCredentialsRepository.java

```diff
@@ -62,7 +62,7 @@ private void checkDependencies() {
             "Missing required dependencies. "
                 + " kadmin are missing or are not found on PATH."
                 + "Possible reason: "
-                + result.getOutput().getUTF8()
+                + (result != null ? result.getOutput().getUTF8() : "")
```

Comment:
can getOutput return null?

Reply:
No... It can throw an IllegalStateException, but it cannot return a null.